src/components/molecules/SfOptions/SfOptions.html

```diff
@@ -0,0 +1,37 @@
+<div class="sf-options">
+  <sf-label
+    v-if="label"
+    :class="['sf-options__title', labelClass]"
+  >
+    {{ label }}
+  </sf-label>
+
+  <div
+    class="sf-options__wrapper"
+    tabindex="0"
+  >
+    <div
```

Comment:
Every option (text, color, image) markup should be wrapped in a separate slot so people can change templates for each of them if they don't like the default one. See [customization doc](https://github.com/DivanteLtd/storefront-ui/blob/master/docs/customization.md) for reference

Reply:
Oh sorry, I have missed from guide. I will fix it soon.

Reply:
I have added slot for options and will add [Scoped Slots](https://vuejs.org/v2/guide/components-slots.html#Scoped-Slots) for access option data.